common/ramdisk/init.sh

```diff
@@ -120,6 +120,7 @@ if [ $ADDITIONAL_CMD_OPTION != "noacs" ]; then
  cat /etc/os-release > /mnt/acs_results/linux_dump/cat-etc-os-release.log
  date > /mnt/acs_results/linux_dump/date.log
  # timedatectl > /mnt/acs_results/linux_dump/timedatectl.txt
```

Comment:
The line of timedatectl can be removed as it is replaced by "cat /proc/driver/rtc"

Reply:
done